Sources/StreamChat/WebSocketClient/Events/UserEvents.swift

```diff
@@ -116,16 +116,34 @@ struct UserWatchingEventDTO: EventWithPayload {
 
 // MARK: - User Ban
 
-public struct UserGloballyBannedEvent: UserSpecificEvent {
-    var userId: UserId
-    var createdAt: Date?
-    var payload: Any
+/// Triggered when user is banned not in a specific channel but globally.
+public struct UserGloballyBannedEvent: Event {
```

Comment:
Not used in SDK, currently. When `user.banned` comes without `cid` - that's a `global` ban.